notebooks/Model_Training_with_BERT.ipynb

```diff
@@ -1420,7 +1420,7 @@
        "      <td>&lt;NA&gt;</td>\n",
```

Comment:
<p>Whoa, I thought my 18min training was slow :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/129/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
<p>Usually these models are closer to 20 minutes. sklearn must have picked a bad initial solution.</p>